tests/integration/container_test.py

```diff
@@ -632,24 +632,24 @@ def test_logs_tail_option(self):
         exitcode = self.client.wait(id)
         self.assertEqual(exitcode, 0)
         logs = self.client.logs(id, tail=1)
-        self.assertEqual(logs, ('Line2\n').encode(encoding='ascii'))
+        self.assertEqual(logs, 'Line2\n'.encode(encoding='ascii'))
 
-#     def test_logs_streaming(self):
-#         snippet = 'Flowering Nights (Sakuya Iyazoi)'
-#         container = self.client.create_container(
-#             BUSYBOX, 'echo {0}'.format(snippet)
-#         )
-#         id = container['Id']
-#         self.client.start(id)
-#         self.tmp_containers.append(id)
-#         logs = bytes() if six.PY3 else str()
-#         for chunk in self.client.logs(id, stream=True):
-#             logs += chunk
+    def test_logs_streaming(self):
+        snippet = 'Flowering Nights (Sakuya Iyazoi)'
+        container = self.client.create_container(
+            BUSYBOX, 'echo {0}'.format(snippet)
+        )
+        id = container['Id']
+        self.client.start(id)
+        self.tmp_containers.append(id)
```

Comment:
minor: I think adding it to tmp_containers should happen before self.client.start(), otherwise start failures would leave a created container lying around